lang/en.js

```diff
@@ -368,6 +368,7 @@ export default {
   "newWindow": "opens in new window",
   "blog": {
     "blog": "Blog",
+    "label": "Blog",
```

Comment:
Instead of adding new keys, we should use the pluralisation format, as we have for the facet filter labels like "Colour | Colours"